case/cheapino-bottom-right.scad

```diff
@@ -10,7 +10,7 @@ step = 0.05;
 // 2.5 will leave 0.35 at the thinnest(between hotswap and magsafe).
 
 
-$fn=$preview ? 50 : 50;
+$fn=50;
```

Comment:
Please use conditional. Take a look at https://github.com/tompi/cheapino/pull/118/files, as @tompi gives a green signal about this https://github.com/tompi/cheapino/issues/72#issuecomment-2495504460.